argoverse/utils/sim2.py

```diff
@@ -35,6 +35,24 @@ def __init__(self, R: np.ndarray, t: np.ndarray, s: Union[int, float]) -> None:
         self.t_ = t.astype(np.float32)
         self.s_ = float(s)
 
+    @property
+    def theta_deg(self) -> float:
+        """Recover the rotation angle `theta` (in degrees) from the 2d rotation matrix.
+
+        Note: the first column of the rotation matrix R provides sine and cosine of theta,
+            since R is encoded as [c,-s]
+                                  [s, c]
+
+        We use the following identity: tan(theta) = s/c = (opp/hyp) / (adj/hyp) = opp/adj
+        """
+        c, s = self.R_[0, 0], self.R_[1, 0]
+        theta_rad = np.arctan2(s, c)
+        return np.rad2deg(theta_rad)
+
+    def __repr__(self) -> str:
+        """Return a human-readable string representation of the class."""
+        return f"Angle (deg.): {self.theta_deg}, Trans.: {np.round(self.t_,2)}, Scale: {self.s_}"
```

Comment:
Can you add the units for `Trans` and `Scale` to be consistent?

Reply:
Thanks for the review, @tagarwal-argoai. So technically, translation (and thus its counterpart scale) are unit-less, as it is up to the user what they choose. It could be inches, meters, centimeters, etc. So I don't think we can include it.

Reply:
I see, thanks for the explanation!